docs/usage.md

```diff
@@ -654,23 +654,6 @@ model = do
 
 is really an unnormalized measure, rather than a probability distribution. `normalize` views it as of type `Weighted Integrator Double`, which is isomorphic to `(Double -> (Double, Log Double) -> Double)`. This can be used to compute the normalization constant, and divide the integrator's output by it, all within `Integrator`. 
 
```

Comment:
Why you do not want to include "Quadrature" in this version of the documentation?

Reply:
It was duplicated.